latexmkrc

```diff
@@ -7,3 +7,6 @@ $pdf_mode = 1;
 
 ## Treat warnings as errors
 $warnings_as_errors = 1;
+
+## Allow many compilation runs
+$max_repeat=10;
```

Comment:
Why are we setting limit to 10? Isn't that too high?

Reply:
It was previously set to 5 but the traceability matrix in particular can take several LaTeX runs to converge to a stable output. In terms of speed, it should not matter too much; only the first run is expensive, the other runs reuse cached information and finish significantly faster.